core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -1620,8 +1620,9 @@ public void importTable(String tableName, Set<String> importDirs)
   }
 
   /**
-   * Prevent potential CRLF injection into logs from read in user data See
-   * https://find-sec-bugs.github.io/bugs.htm#CRLF_INJECTION_LOGS
+   * Prevent potential CRLF injection into logs from read in user data.
+   *
+   * @see <a href=https://find-sec-bugs.github.io/bugs.htm#CRLF_INJECTION_LOGS/>
```

Comment:
In my IDE, there is nothing to click to follow the here:
![image](https://user-images.githubusercontent.com/47725857/170050074-c0f81df8-5549-486e-a1be-bb6bda4bbb90.png)
Maybe "CRLF injection" in the sentence could be linked or some other text to click to follow the link.